.github/workflows/test-setup-minconda.yml

```diff
@@ -41,9 +41,9 @@ jobs:
         run: |
           ${CONDA_RUN} python --version
           ${CONDA_RUN} python --version | grep "${PYTHON_VERSION}"
-      - name: Can use ${CONDA_INSTALL}, installs requests
+      - name: Can use ${CONDA_INSTALL}, installs some conda packages
         run: |
-          ${CONDA_INSTALL} requests
+          ${CONDA_INSTALL} numpy cmake cffi ninja typing_extensions dataclasses pip
```

Comment:
did you deliberately remove "requests" from this list?

Reply:
Ah, conda already a requests installed, so this command doesn't really check for anything.  I intentionally install some packages required by PyTorch here to make sure that they are installed ok